deploy/operator.yaml

```diff
@@ -2,23 +2,23 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: nmstate-handler
-  namespace: {{ .Namespace }}
+  name: {{with .HandlerPrefix}}{{printf "%s-" .}}{{end}}nmstate-handler
```

Comment:
Can we use a golang template var at the top of the template with the `with` case ? https://golang.org/pkg/text/template/#hdr-Variables